bleak/backends/p4android/client.py

```diff
@@ -273,7 +273,7 @@ async def get_services(self) -> BleakGATTServiceCollection:
                     java_characteristic,
                     service.uuid,
                     service.handle,
-                    self.__mtu - 3,
+                    lambda: self.__mtu - 3,
```

Comment:
Off topic, but it reminds me that it would be nice get rid of name-mangling, IMO.  I guess it's only the android client though.